recipes/r-recetox-aplcms/meta.yaml

```
@@ -22,36 +22,48 @@ build:
   
 requirements:
   host:
-    - r-base
+    - r-base >=3.5
     - r-mass
     - r-rgl
-    - bioconductor-mzr
+    - bioconductor-mzR
+    - r-splines2
     - r-doparallel
     - r-foreach
     - r-iterators
     - r-snow
     - r-gbm
     - r-e1071
-    - r-randomforest
+    - r-randomForest
```

```suggestion
    - r-randomforest
```
All R packages are lowercase. Doesn't seem to make a functional difference, but it's incorrect nontheless.